src/cmds/restic/cmd_restore.go

```diff
@@ -13,6 +14,8 @@ type CmdRestore struct {
 	Exclude []string `short:"e" long:"exclude" description:"Exclude a pattern (can be specified multiple times)"`
 	Include []string `short:"i" long:"include" description:"Include a pattern, exclude everything else (can be specified multiple times)"`
 	Target  string   `short:"t" long:"target"  description:"Directory to restore to"`
+	Source  string   `short:"s" long:"source"  description:"Source Filter (for id=latest)"`
```

Comment:
I think the name `Source` is misleading. What about `host` or `hostname`?

Of course, this also applies to the heading of the table printed for `snapshots`.